README_gr-GR.md

```diff
@@ -0,0 +1,452 @@
+# Οδηγός για τα "Μηνύματα Commit"
+
+[![Πείτε ευχαριστώ!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+Αυτό το κείμενο αποτελεί έναν οδηγό που θα συντελέσει στο να κατανοήσετε την σημασία των μηνυμάτων commit και πώς να τα γράψετε σωστά.
+
+Ενδεχομένως να σας βοηθήσει να μάθετε τι ακριβώς είναι ένα commit, γιατί θεωρείται σημαντικό το να γράφετε καλά μηνύματα, βέλτιστες πρακτικές και κάποιες συμβουλές για το πώς να προσχεδιάσετε και (ξανά)συντάξετε ένα καλό ιστορικό commit.
+
+## Διαθέσιμες Γλώσσες
```

Comment:
Since #31 this section is not needed anymore in translations (see [this](https://github.com/xlxs4/commit-messages-guide/blob/greek-patch/CONTRIBUTING.md#new-translation-checklist) CONTRIBUTING.md section).

I'll merge as is and I remove it in a new commit, don't worry.